static/src/stylesheets/module/_release-message.scss

```diff
@@ -466,6 +466,81 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
     background: colour(membership-default);
 }
 
+
+
+/*
+ * Coin based adblock message
+ */
+.adblock-sticky__message.adblock-sticky__coin {
```

Comment:
Why are we using this combinator? BEM class names are supposed to enact namespacing by themselves. Was this needed for specificity?